Fixes 
ld: warning: symbol 'facilitynames' has differing sizes:
        (file /usr/lib/libplumb.so value=0xb0; file /usr/lib/libqb.so value=0xa8);
        /usr/lib/libplumb.so definition taken

--- cluster-glue-bdd95fd0f89f31b19fa46ffa0ea3f5022ebcd858/lib/clplumbing/cl_syslog.c.1	2017-09-17 09:26:13.802105258 +0000
+++ cluster-glue-bdd95fd0f89f31b19fa46ffa0ea3f5022ebcd858/lib/clplumbing/cl_syslog.c	2017-09-17 09:26:19.869679768 +0000
@@ -48,7 +48,6 @@
 {
 #ifdef LOG_AUTH
 	{ "auth", LOG_AUTH },
-	{ "security", LOG_AUTH },           /* DEPRECATED */
 #endif
 #ifdef LOG_AUTHPRIV
 	{ "authpriv", LOG_AUTHPRIV },
